pages/outreachy.md

```diff
@@ -54,8 +57,56 @@ Interested in becoming a mentor? Contact one of the coordinators!
 
 ## Available tasks
 
-Coming soon. At this moment, we recommend looking at the following issue trackers for smaller tasks that might
-give a sense of what's coming:
+### Instrumentation for mobile applications
+
+Mentor: Gary Brown
+
+* Collect ideas on what's possible and what's desirable for mobile applications
+* Decide and experiment with different options for an optimized mobile experience, like, which sampling strategies
+make more sense
+* Create the missing pieces, like OpenTracing framework integration for some Android component, or a specific Jaeger
+tracer for mobile usage
+* Documentation + examples
+* The end result would be: a trace starts when a user opens the app on the phone and backend microservices are called
+
+### Split the JavaScript OT library: NodeJS and Browser
```

Comment:
This is a jaeger specific task I believe @pavolloffay ?

Reply:
Indeed it is!

Reply:
Maybe add Jaeger in the title